### PR TITLE
feat: Workers and jobs tables with realtime data.

### PR DESCRIPTION
**Pull request by amberolive**
Please review.

### CODE REVIEW

File: app/components/nav-bar/template.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- Navbar -->
+    <nav class="navbar navbar-inverse">
```
Comment (donaldnash1989) [inline app/components/nav-bar/template.hbs]: Are we keeping this as the basic inverted or are we going to have it fit the KUB color scheme?

File: app/components/jobs-table/template.hbs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    </tr>
+                </thead>
+                <tbody>
+                    {{yield}}
+                </tbody>
+            </table>
+        </div>
+        <div class="panel-footer">
+            <div class="row">
+                <div class="col col-sm-4 col-md-4 col-lg-4">Page 1 of 5
```
Comment (donaldnash1989) [inline app/components/jobs-table/template.hbs]: Even though its a small detail, we can't forget about this pagination later.

Comment (amberolive) [inline app/components/jobs-table/template.hbs]: Yes. I added a card for it to Trello.

File: app/components/nav-bar/template.hbs (hunk 1 above)
Comment (RoundDev) [inline app/components/nav-bar/template.hbs]: It is up to you guys. Personally, I prefer darker but to be consistent with the design we may have to change.

**Pull request merged**